specification/storageactions/StorageAction.Management/tspconfig.yaml

```diff
@@ -45,6 +45,10 @@ options:
     generate-fakes: true
     head-as-boolean: true
     inject-spans: true
+  "@azure-typespec/http-client-csharp-mgmt":
+    package-name: "Azure.ResourceManager.StorageActions"
+    namespace: "{package-name}"
+    new-project: false
```

Comment:
will it work without this? thought it should be default to false

Reply:
I think so, let me remove this and have a try